alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -62,46 +75,51 @@ data class Alert(
          * Sets the [title] displayed in the alert
          *
          * @param title The title of the alert
+         * @return The modified [Alert.Builder]
          */
         fun setTitle(title: String?) = apply { this.title = title }
 
         /**
          * Sets the [message] displayed in the alert
          *
          * @param message The message of the alert
+         * @return The modified [Alert.Builder]
          */
         fun setMessage(message: String?) = apply { this.message = message }
 
         /**
-         * Sets button with the id `BUTTON_POSITIVE` on Android
-         * and action with style `UIAlertActionStyleDefault` on iOS
+         * Sets an [Action] with [Action.Style.POSITIVE]. If one already exists it will be replaced.
          *
          * @param title The title of the button
          * @param handler The block to execute after user taps a button
+         * @return The modified [Alert.Builder]
          */
         fun setPositiveButton(title: String, handler: AlertActionHandler = {}) = apply {
+            this.actions.removeAll { it.style == Action.Style.POSITIVE || it.style == Action.Style.DEFAULT }
```

Comment:
What if I need two positive buttons? I think it should be a flag passed to remove existing one